Backend/Models/Configurations/PharmacyConfiguration.cs

```diff
@@ -11,6 +11,10 @@ public void Configure(EntityTypeBuilder<Pharmacy> builder)
             builder
                 .HasMany(p => p.Registers)
                 .WithOne(r => r.Pharmacy);
+
+            builder
+                .HasMany(r => r.RequiredMedicamentAmounts)
+                .WithOne(p => p.Pharmacy);
```

Comment:
Same as in `MedicamentConfiguration.cs`.

Reply:
Fixed